### ISSUE TITLE
[BUG] Menu Bar Disappear

### ISSUE THREAD

**Issue by tyramsihu** [Paperback-iOS/app#847]
### Preflight Checklist

- [X] I have read and I agree to follow the [support disclaimer](https://github.com/Paperback-iOS/app#support-disclaimer).
- [X] I am sure that my bug has nothing to with third party extensions.
- [X] I have read the [troubleshooting FAQs](https://paperback.moe/help/faq/) and looked trough the \#help-desk channel on the [official Discord](https://discord.com/invite/Paperback), but none solved my issue.
- [X] I have searched the [issue tracker](https://github.com/Paperback-iOS/app/issues) for a bug report that matches the one I want to file, without success.

### What type of Apple device have you encountered the bug with?

iPad

### Operating System Version

iPadOS 18

### App Version

0.8.8-r.31

### Describe the Bug

After finishing update my iPad to iPadOS 18, the menu bar/navigation on Paperback disappear, it become an island like navigation above with no name and icon.
![IMG_6200](https://github.com/user-attachments/assets/9cdd01b7-dfe8-49fc-abc1-6ae6a37882cd)

### Steps to Reproduce

update to ios/iPadOS 18

**Comment by tyramsihu** [comment]
how to install it? i'm new to this

**Comment by tyramsihu** [comment]
> I think this will help you.
> https://mega.co.nz/#!9vE0jIrD!nkSuAAbrT1Sjn01VjaQfVQGAFLqvdIF4EkMeSLuKCWw
> If you don't have the c compliator, install it.(gcc or clang)
> 

how to install it? i'm new to this

**Comment by Celarye** [comment]
@tyramsihu That was a scam, ignore it. The bug you reported seems to be something @FaizanDurrani will need to take a look at.

**Comment by tyramsihu** [comment]
> @tyramsihu That was a scam, ignore it. The bug you reported seems to be something @FaizanDurrani will need to take a look at.

I see, almost get scammed, btw thank you, looking forward for the update

**Comment by just-for-death** [comment]
Any updates on any fix

**Comment by Celarye** [comment]
A build with a fix should release the coming days iirc.

**Comment by Catta1997** [comment]
> A build with a fix should release the coming days iirc.

if can help, on 0.8.7(12) there is no issue

**Comment by Celarye** [comment]
That’s correct, that is because that version was build with the iOS 17 SDK, while the new has been build with the new iOS 18 SDK.

Though be careful with using the old version as that had some major bugs which could brick the app.

**Comment by tyramsihu** [comment]
is there any update on this issue?

**Comment by Celarye** [comment]
@FaizanDurrani was this fixed with 0.8.8 or has it not been released yet?

**Comment by denkristoffer** [comment]
The menu bar is back with the latest release 👍
